src/main/java/org/mitre/synthea/world/geography/Location.java

```diff
@@ -270,8 +270,8 @@ public void assignPoint(Person person, String cityName) {
       // Precision within 0.001 degree is more or less a neighborhood or street.
       // Precision within 0.01 is a village or town
       // Precision within 0.1 is a large city
-      double dx = person.rand(0.001, 0.1);
-      double dy = person.rand(0.001, 0.1);
+      double dx = person.rand(-0.05, 0.05);
+      double dy = person.rand(-0.001, 0.05);
```

Comment:
Is there a reason why `dy` isn't between -.05 and .05 like `dx`?

Reply:
Ugh. Sorry. I was playing with things and didn't reset. I'll fix that.